quantum/keycode_config.c

```diff
@@ -108,6 +128,24 @@ uint8_t mod_config(uint8_t mod) {
             mod |= MOD_RGUI;
         }
     }
+    if (keymap_config.swap_lctl_lgui) {
+      if ((mod & MOD_RGUI) == MOD_LGUI) {
+        mod &= ~MOD_LGUI;
+        mod |= MOD_LCTL;
+      } else if ((mod & MOD_RCTL) == MOD_LCTL) {
+        mod &= ~MOD_LCTL;
```

Comment:
it feels like these two lines should be a more descriptive macro, like CANCEL_AND_SET_MOD(MOD_LCTL, MOD_LGUI)

Reply:
do you mean for the whole function?   
If so, the function is from tmk originally, and is used extensively, IIRC. 

Otherwise, we don't actually have a function to modify mods in a variable.  So, something would need to be added, to do so.

Reply:
nope, just the inner block. It would be clearer what's going on.

Reply:
Okay.   unfortunately, no function exists for that right now.  But it could be useful. 

For now, though, I don't think this is a change that needs to be added here.   It's something that would be better in a more general-ish overhaul.